README.md

```diff
@@ -67,10 +69,25 @@ You will now have a local clone of the remote repository as an Rstudio project.
 The same directories and files which can be seen on the remote (https://github.com/inbo/protocols) will be copied to your local drive. 
 Whenever you want to work in the project, you need to double-click `protocols.Rproj` file to start the Rstudio project. 
 
-However, multiple Rstudio projects are associated with subfolders beneath the root directory. 
-These projects will automatically be associated with the same `git` version control as the `protocols.Rproj`. 
+To refer to files use the `rprojroot` package. Here is an example to refer to the utils folder:
 
-Branching model:
+```r
+library(rprojroot)
+path_to_utils <- file.path(
+  find_root(is_git_root),
+  "src",
+  "utils"
+  )
+```
+
+
+*Branching model:*
+
+```
+  |- master branch <- contains only finished and approved protocols, both source code and html rendered github pages
+      | - develop branch <- contains development versions of protocols, this is the default branch
```

Comment:
The `master` and `releases` suggestion is to my mind more difficult as it departs from most other repository setups. But I may be convinced otherwise...